app/components/FilterableTable/Filters/EnumFilter.tsx

```diff
@@ -8,7 +8,7 @@ export default class EnumFilter<T> extends TableFilter {
   enumValues: T[];
 
   constructor(display, argName, enumValues: T[]) {
-    super(display, argName);
+    super(display, argName, {sortable: false});
```

Comment:
This is actually breaking the functional sort option on all the other tables using an `EnumFilter`, we just want to disable it for this particular field [here](https://github.com/bcgov/cas-ciip-portal/blob/develop/app/containers/Facilities/FacilitiesListContainer.tsx#L32)

Reply:
ah OK, wasn't sure if some enums should be sorted. I think I'll have to export this `ISearchOptionSettings` interface then since there's a few layers of inheritance going on here.